src/renderers/js-experimental/nameTransformers.ts

```diff
@@ -147,6 +148,8 @@ export const DEFAULT_NAME_TRANSFORMERS: NameTransformers = {
   programInstructionsEnumVariant: (name) => `${pascalCase(name)}`,
   programInstructionsIdentifierFunction: (name) =>
     `identify${pascalCase(name)}Instruction`,
+  programInstructionsTypeUnion: (name) =>
```

Comment:
nit: I'd try and stay consistent with the fragment function name here.

```suggestion
  programInstructionsParsedUnionType: (name) =>
```